src/openrct2-ui/windows/Options.cpp

```diff
@@ -1007,26 +1028,9 @@ class OptionsWindow final : public Window
 
     void DisplayDraw(rct_drawpixelinfo* dpi)
     {
-        DrawTextBasic(
-            dpi, windowPos + ScreenCoordsXY{ 10, widgets[WIDX_FULLSCREEN].top + 1 }, STR_FULLSCREEN_MODE, {}, { colours[1] });
-
-        // Disable resolution dropdown on "Windowed" and "Fullscreen (desktop)"
-        colour_t colour = colours[1];
-        if (gConfigGeneral.fullscreen_mode != static_cast<int32_t>(OpenRCT2::Ui::FULLSCREEN_MODE::FULLSCREEN))
-        {
-            colour |= COLOUR_FLAG_INSET;
-        }
-        DrawTextBasic(
-            dpi, windowPos + ScreenCoordsXY{ 10 + 15, widgets[WIDX_RESOLUTION].top + 1 }, STR_DISPLAY_RESOLUTION, {},
-            { colour });
-        DrawTextBasic(
-            dpi, windowPos + ScreenCoordsXY{ 10, widgets[WIDX_SCALE].top + 1 }, STR_UI_SCALING_DESC, {}, { colours[1] });
-        DrawTextBasic(
-            dpi, windowPos + ScreenCoordsXY{ 10, widgets[WIDX_DRAWING_ENGINE].top + 1 }, STR_DRAWING_ENGINE, {},
-            { colours[1] });
-
         auto ft = Formatter();
         ft.Add<int32_t>(static_cast<int32_t>(gConfigGeneral.window_scale * 100));
+        // FIXME: Add a proper string for this instead of using STR_WINDOW_OBJECTIVE_VALUE_RATING.
```

Comment:
Please create an issue for it (using the Blank issue template), rather than putting in 3 FIXMEs.

Reply:
Sure, done. :^)